packages/tailwindcss/src/utilities.ts

```diff
@@ -4588,3 +4588,8 @@ export function createUtilities(theme: Theme) {
 
   return utilities
 }
+
+export function isPositiveInteger(value: any) {
+  let num = Number(value)
+  return Number.isInteger(num) && num >= 0
+}
```

Comment:
Yeah, agree. I was split between exporting this here or simply copy-pasting it to the other places where I need it since it's so lightweight anyways. `infer-data-types.ts` is a better place for this for now though, I agree!